### DISCUSSION TITLE
sqlalchemy db when deployed to deta changed into readonly

### DISCUSSION THREAD

**Discussion by SaketMunda** [deta/cloud-docs#257] [Help] [answered]
using fastapi and sqlalchemy db, working fine in local env but when deployed to deta micro it changed to readonly and not allowing me to write any objects. Is there any configurations to be done ?

**Comment by cofob** [comment, marked as answer]
[Read the docs](https://docs.deta.sh/docs/micros/about#technical-specifications).

**Comment by SaketMunda** [comment]
I think this is the answer, Micros only support read-only SQLite, which you could deploy with your code.
